lib/app.js

```diff
@@ -136,7 +136,8 @@ const MAPPINGS = {
   editorsChoice: {
     path: ['ds:5', 0, 12, 15, 0],
     fun: Boolean
-  }
+  },
+  available: ['internal:available']
```

Comment:
I'm not convinced by this syntax. If a field doesn't fit with the MAPPINGS structure add it separately in the parsing code (with an explanatory command) but not here, since it's not really meant to be extracted from the same data object